core/src/main/scala/tech/beshu/ror/accesscontrol/domain.scala

```diff
@@ -23,6 +23,7 @@ import cats.kernel.Monoid
 import com.comcast.ip4s.{Cidr, Hostname, IpAddress}
 import eu.timepit.refined.api.Refined
 import eu.timepit.refined.auto._
+//import eu.timepit.refined.types.all.NonEmptyString
```

Comment:
Commented code

Reply:
I'm preparing some refactor related to this task. I promise to fix it in the next PR